test/fixtures/scrub.txt

```diff
@@ -1 +1 @@
-http://whatismyipaddress.com/
+http://geoip.ubuntu.com/lookup
```

Comment:
Use the same service we use for getting the user IP. Rationale: the former service is not blocked. In the event in which `geoip.ubuntu.com` blocks travis, it's unclear whether this test breaks, tho.